modules/tensor_mechanics/src/kernels/GeneralizedPlaneStrainOffDiag.C

```diff
@@ -16,6 +16,7 @@
 #include "MooseVariableScalar.h"
 #include "RankTwoTensor.h"
 #include "RankFourTensor.h"
+#include "UserObjectInterface.h"
```

Comment:
Kernel already derives from and includes user object interface. Is it really necessary to add this include here again?